src/ocamlorg_frontend/pages/platform.eml

```diff
@@ -7,6 +7,16 @@ let render_sidebar
     <%s! Learn_sidebar.render ~tutorials ~current_tutorial:active ~section:Learn_layout.Platform %>
   </div>
 
+let render_package_cards ~(item: Data.Tool.t) =
+  <a href="<%s item.source %>" class="flex gap-4 border-2 rounded-md p-4 h-28 hover:border-primary-600 hover:ring-4 hover:ring-primary-200">
+    <%s! Icons.tools "h-5 w-5 text-primary-600 self-start" %>
```

Comment:
There's something off about the alignment of the icon with respect to the name of the item. In the design, it looks like the orange text and the icon are pretty much center aligned wrt each other. :thinking:

I think the simplest way to achieve this is probably to make the icon and name a flex-row, and then, have another flex-row below (with an empty div on the left that's the same width as the icon.

Reply:
Done. Should be better now